File: Easy/LastWord.js
var lengthOfLastWord = function(s) {
  let res = s.split(' ');
  let result = [];
  
  if (s.length === 0) {
      return 0;
  }
  for (let i = 0; i < res.length; i++) {
      if (res[i] !== '') {
         result.push(res[i]);
      }  
  }
  
  let len = result[result.length - 1];
  return len.length;
};

let a = 'a ';
console.log(a.length);
console.log(lengthOfLastWord(a));

File: Easy/LastWord.py
class Solution:
    def lengthOfLastWord(self, s):
        """
        :type s: str
        :rtype: int
        """
        res = s.split(' ')
        result = []

        if len(s) == 0:
            return 0

        for i in range(len(res)):
            if res[i] != '':
                result.append(res[i])
        length = result[len(result) - 1]
        return len(length)


test = Solution()
s = 'asdasd asda sfaf adadsdasada'
print test.lengthOfLastWord(s)
